config/default-cluster/pipeline-role.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: pipeline-role
```

Comment:
I don't think we should suffix names with the kind like this, can you remove `-role`?